### PR TITLE
#1826 Load child_range into the database

### PR DESCRIPTION
**Pull request by alexander-vo**
child ranges from from the sc-data/structure/child_range.json file will be loaded to the child_range collection in the ArangoDB

### CODE REVIEW

**Comment by alexander-vo** [comment]
Collection state:

image

**Comment by sujato** [comment]
Looks good to me.

File: server/server/data_loader/arangoload.py [hunk 1]
```diff
@@ -720,6 +740,9 @@ def print_stage(msg):
     print_stage("Building and loading navigation from structure_dir")
     add_root_docs_and_edges(change_tracker, db, structure_dir)
 
+    print_stage("Loading child ranges from structure_dir")
+    load_child_range(db, structure_dir)
+
```
Comment (Prograd) [inline server/server/data_loader/arangoload.py:745]: Do we need this empty line? It seems to me like unnecessary change.

File: server/server/migrations/migrations/add_child_range_collection_027.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+from migrations.base import Migration
+
+
+class SecondMigration(Migration):
+    migration_id = 'add_child_range_collection_027'
+    tasks = ['create_collections']
+
+    def create_collections(self):
+        db = get_db()
+
```
Comment (Prograd) [inline server/server/migrations/migrations/add_child_range_collection_027.py:11]: The same here, I guess you can remove it.

File: server/server/data_loader/arangoload.py (hunk 1 above)
Comment (robertmusil) [reply server/server/data_loader/arangoload.py:745]: Why? It adds nice separation of logical blocks.

Comment (Prograd) [reply server/server/data_loader/arangoload.py:745]: If you say so, then it's. I usually avoid such such separator.

File: server/server/migrations/migrations/add_child_range_collection_027.py (hunk 2 above)
Comment (alexander-vo) [reply server/server/migrations/migrations/add_child_range_collection_027.py:11]: Well, yes, you can do it without skipping a line, although it is only used to visually separate logic
but I also don't think it's a big deal to reopen the pull-request for that 
you can also look at other migration files and see that "annoying" line skip there =D
